File: 2019/CSharp/Program.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Reflection;
using AdventOfCode.CSharp.Common;

namespace AdventOfCode.CSharp.Year2019
{
    class Program
    {
        static void Main(string[] args)
        {
            //var xxx = new Day15_Old.Day15Puzzle1();
            //xxx.Run();

            var runner = new PuzzleRunner(
                "Advent of Code 2019",
                "https://adventofcode.com/2019",
                "By: SDragon");
            runner.Run(args);
        }

    }
}

File: 2019/FSharp/Program.fs
// Learn more about F# at http://docs.microsoft.com/dotnet/fsharp

open System
open FSharp.Common

[<EntryPoint>]
let main args =

    let runner = PuzzleRunner([|
        "Advent of Code 2019";
        "https://adventofcode.com/2019";
        "By: SDragon"|])
    
    runner.Run(args)

    0 // return an integer exit code
